src/lib/swapper/swappers/ThorchainSwapper/getThorTradeQuote/getTradeQuote.ts

```diff
@@ -177,21 +176,13 @@ export const getThorTradeQuote = async (
     return bnOrZero(expectedAmountOutThorBaseUnit).div(sellAmountCryptoThorBaseUnit).toFixed()
   }
 
-  const getRouteBuyAmount = (expectedAmountOutThorBaseUnit: string) => {
-    // Because the expected_amount_out is the amount after fees, we need to add them back on to get the "before fees" amount
-    // Add back the outbound fees
-    const expectedAmountPlusFeesCryptoThorBaseUnit = bn(expectedAmountOutThorBaseUnit)
-      .plus(fees.outbound)
-      .plus(fees.affiliate)
-
-    return toBaseUnit(
-      fromBaseUnit(expectedAmountPlusFeesCryptoThorBaseUnit, THORCHAIN_FIXED_PRECISION),
-      buyAsset.precision,
-    )
+  const getRouteBuyAmount = (quote: ThornodeQuoteResponseSuccess) => {
+    const emitAsset = bn(quote.expected_amount_out).plus(quote.fees.outbound)
```

Comment:
update to just calculate `emit_asset` from the quote directly instead of trying to infer what calculations were applied to `expectedAmountPlusFeesCryptoThorBaseUnit`